### ISSUE TITLE
loss issue

### ISSUE THREAD

**Issue by blue-butterfly** [deepinsight/insightface#367]
I want to print the both the accuracy and loss，the accuracy is normal, but loss value is very very small , why?

**Comment by nttstar** [comment]
why what?

**Comment by blue-butterfly** [comment]
thanks for reply!  I print the accuracy and loss ,  the loss is very small from beginning. I don't know the reason.

INFO:root:Epoch[0] Batch [20]	Speed: 599.97 samples/sec	acc=0.000279	lossvalue=0.000094
INFO:root:Epoch[0] Batch [40]	Speed: 611.21 samples/sec	acc=0.003711	lossvalue=0.000090
INFO:root:Epoch[0] Batch [60]	Speed: 643.76 samples/sec	acc=0.007910	lossvalue=0.000093
INFO:root:Epoch[0] Batch [80]	Speed: 679.45 samples/sec	acc=0.012012	lossvalue=0.000079
INFO:root:Epoch[0] Batch [100]	Speed: 634.18 samples/sec	acc=0.020020	lossvalue=0.000071
INFO:root:Epoch[0] Batch [120]	Speed: 643.40 samples/sec	acc=0.023340	lossvalue=0.000069
INFO:root:Epoch[0] Batch [140]	Speed: 608.34 samples/sec	acc=0.033398	lossvalue=0.000071
INFO:root:Epoch[0] Batch [160]	Speed: 672.80 samples/sec	acc=0.041406	lossvalue=0.000074
INFO:root:Epoch[0] Batch [180]	Speed: 631.44 samples/sec	acc=0.044141	lossvalue=0.000072
INFO:root:Epoch[0] Batch [200]	Speed: 683.48 samples/sec	acc=0.054883	lossvalue=0.000064
INFO:root:Epoch[0] Batch [220]	Speed: 627.80 samples/sec	acc=0.069238	lossvalue=0.000065
INFO:root:Epoch[0] Batch [240]	Speed: 564.42 samples/sec	acc=0.069922	lossvalue=0.000076
INFO:root:Epoch[0] Batch [260]	Speed: 606.62 samples/sec	acc=0.079590	lossvalue=0.000058
INFO:root:Epoch[0] Batch [280]	Speed: 650.64 samples/sec	acc=0.086035	lossvalue=0.000067
INFO:root:Epoch[0] Batch [300]	Speed: 595.01 samples/sec	acc=0.095801	lossvalue=0.000050
INFO:root:Epoch[0] Batch [320]	Speed: 562.29 samples/sec	acc=0.110742	lossvalue=0.000058
INFO:root:Epoch[0] Batch [340]	Speed: 606.04 samples/sec	acc=0.120313	lossvalue=0.000048
INFO:root:Epoch[0] Batch [360]	Speed: 677.55 samples/sec	acc=0.128906	lossvalue=0.000043
INFO:root:Epoch[0] Batch [380]	Speed: 627.69 samples/sec	acc=0.140820	lossvalue=0.000045
INFO:root:Epoch[0] Batch [400]	Speed: 669.58 samples/sec	acc=0.149707	lossvalue=0.000039
INFO:root:Epoch[0] Batch [420]	Speed: 603.19 samples/sec	acc=0.163965	lossvalue=0.000037
INFO:root:Epoch[0] Batch [440]	Speed: 611.14 samples/sec	acc=0.170703	lossvalue=0.000036

**Comment by blue-butterfly** [comment]
I  use the class LossValueMetric of your train_softmax.py， and revise the code of train_net function as:
    _metric = []
    if args.loss_type<10:
           _metric_acc = AccMetric()
          _metric_loss = LossValueMetric()
          _metric.append(_metric_acc)
          _metric.append(_metric_loss)
    else:
           _metric_loss = LossValueMetric()
           _metric.append(_metric_loss)
     eval_metrics = [mx.metric.create(_metric)]

**Comment by nttstar** [comment]
Not correctly used. Training symbols do not output loss value.

**Comment by blue-butterfly** [comment]
thanks，I am sorry I only begin to learn mxnet. Do you mean the class   LossValueMetric do not get the real loss，I need to realize by by myself？

**Comment by nttstar** [comment]
I will add this feature later.

**Comment by blue-butterfly** [comment]
Thanks very much!!!!!!@nttstar

**Comment by nttstar** [comment]
use ``--ce-loss`` in command line option.
